src/main/resources/validationrules/core/hearing-role-caseworker-global.drl

```diff
@@ -29,7 +29,8 @@ import function uk.gov.hmcts.reform.orgrolemapping.util.UtilityFunctions.getJuri
    $up: CaseWorkerAccessProfile((roleId in ("1","2","16") && serviceCode in ("BBA3"))
                                   || (roleId in ("1","2") && serviceCode in ("ABA3","ABA5"))
                                   || (roleId in ("1","2") && serviceCode in ("AAA6","AAA7"))
-                                  || (roleId in ("1","2") && serviceCode in ("BHA1")),
+                                  || (roleId in ("1","2") && serviceCode in ("BHA1"))
+                                  || (roleId in ("1","2") && serviceCode in ("BFA1")),
```

Comment:
can this be coded like this 
|| (roleId in ("1","2") && serviceCode in ("BHA1,"BFA1""))

Reply:
Same for rest of the below lines

Reply:
I have separated them out as the service codes are for different services `(BHA1 - ET, BFA1 - IA)`. Whereas the one above `(roleId in ("1","2") && serviceCode in ("AAA6","AAA7"))` the 2 service codes belong to the same CIVIL jurisdiction.

Reply:
I agree with @abul-syed that it is useful to keep the service config on separate lines.